modules/store-devtools/src/instrument.ts

```diff
@@ -64,12 +64,16 @@ export function noMonitor(): null {
   return null;
 }
 
+export const DEFAULT_NAME = 'ngrx-store-devtool-instance';
```

Comment:
Change to `NgRx Store DevTools`

Reply:
I changed the name to NgRx Store DevTools